src/boss/boss_translator.erl

```diff
@@ -50,6 +50,12 @@ reload_all(Pid) ->
 %% @spec fun_for(Locale::string()) -> TranslationFun::function() | none
 fun_for(Pid, Locale) ->
     case is_loaded(Pid, Locale) of
-        true -> fun(String) -> ?MODULE:lookup(Pid, String, Locale) end;
+        true ->
+            fun(String) ->
+                case ?MODULE:lookup(Pid, String, Locale) of
+                    undefined -> String;
```

Comment:
@brigadier I'm not quite sure if in general it's not better to stick with `undefined`, as I think that from the point of using the interface it would maybe be helpful for someone to check if a translation was found or not. This change would also break backward compatibility. We also had the need for that and implemented a wrapper around the `lookup/3` method:
```erl
handle_translation(TextToTranslate) ->
    TPid = boss_web:translator_pid(app_config:app_name()),
    case boss_translator:lookup(TPid, TextToTranslate, <locale>) of
        undefined ->
            TextToTranslate;
        Translation ->
            Translation
    end.
```
Maybe it would be better to introduce some additional functions for example `fun_for_<foo>/2` and for `lookup_<foo>/3` which could provide this behaviour and also keep backward compatibility?